Bump pypck to 0.6.1

alengwenus opened the pull request:
## Description:
Bump to pypck==0.6.1
This is used by the LCN component.

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [x] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [x] Untested files have been added to `.coveragerc`.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

alengwenus committed:
Bump to pypck==0.6.1

OttoWinter commented:
Wait, the changelog [here](https://github.com/alengwenus/pypck/compare/release-0.6.0...release-0.6.1) shows that only the version number got bumped - did you forget to `git add` the files for "Removed unused files from distribution"?

alengwenus commented:
> Wait, the changelog [here](https://github.com/alengwenus/pypck/compare/release-0.6.0...release-0.6.1) shows that only the version number got bumped - did you forget to `git add` the files for "Removed unused files from distribution"?

Nope... I renamed some files in a previous version but forgot to delete the old ones from the build directory. So they where packed and uploaded to pypi, although they were used nowhere. Now, I just deleted the old files... Actually the code did not change but pypi forced me to generate a new version number.
